lib/src/configuration.dart

```diff
@@ -131,6 +131,8 @@ class Configuration {
     commits = configValues.commits ?? 'auto';
     ignoreMissing = configValues.ignoreMissing ?? false;
 
+    buildFilesFolder =
+        configValues.buildFilesFolder ?? _fs.currentDirectory.path;
```

Comment:
So if I understand correctly if we don't specify a folder then it just takes the current path, maybe we can also document that somewhere in the readme

Reply:
oh yeah, missed that, all good then